Makefile

```diff
@@ -8,8 +7,17 @@ ESLINT=npx eslint
 # through setting FLAGS environment variable
 FLAGS:=$(if $(FLAGS),$(FLAGS),"--config=config.yaml")
 
-SUPERVISORD:=PYTHONPATH=. FLAGS=$$FLAGS python -m supervisor.supervisord -c baselayer/conf/supervisor/supervisor.conf
-SUPERVISORCTL:=PYTHONPATH=. FLAGS=$$FLAGS python -m supervisor.supervisorctl -c baselayer/conf/supervisor/supervisor.conf
+PYTHON=PYTHONPATH=. python
+ENV_SUMMARY=$(PYTHON) baselayer/tools/env_summary.py $(FLAGS)
+
+# Flags are propagated to supervisord via the FLAGS environment variable
+# Inside of supervisord configuration files, you may reference them using
+# %(ENV_FLAGS)s
+SUPERVISORD_CFG=baselayer/conf/supervisor/supervisor.conf
+SUPERVISORD=export FLAGS=$(FLAGS) && $(PYTHON) -m supervisor.supervisord -c $(SUPERVISORD_CFG)
+SUPERVISORCTL=$(PYTHON) -m supervisor.supervisorctl -c $(SUPERVISORD_CFG)
+
+LOG=@$(PYTHON) -c "from baselayer.log import make_log; spl = make_log('skyportal'); spl('$1')"
```

Comment:
Is this skyportal-specific reference intentional?

Reply:
Should be baselayer, thanks.